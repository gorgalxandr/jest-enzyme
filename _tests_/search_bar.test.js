import React from 'react';
import ReactDOM from 'react-dom';
import SearchBar from '../src/components/search_bar';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import {shallow} from 'enzyme';

describe('Component: App', () => {


  it('should match its empty index snapshot', () => {
    const tree = renderer.create(<SearchBar />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
