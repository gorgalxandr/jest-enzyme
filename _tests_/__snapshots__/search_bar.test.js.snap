// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: App should match its empty index snapshot 1`] = `
<div
  className="search-bar"
>
  <input
    onChange={[Function]}
    value=""
  />
</div>
`;
